src/StringSorterDescTest.java

```diff
@@ -0,0 +1,12 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+class StringSorterDescTest {
+
+    @org.junit.jupiter.api.Test
+    void sortStringsDesc() {
+        String[] testStrings = {"cow", "bat", "dog", "zebra", "llama"};
+        String[] manuallySorted = {"zebra", "llama", "dog", "cow", "bat"};
+        String[] sortedDescArray = StringSorterDesc.sortStringsDesc(testStrings);
+        assertArrayEquals(manuallySorted, sortedDescArray);
```

Comment:
This is a great way to test, you could also use the Assert library included in Junit to minimize overhead but using the Jupiter API works great too!